neuralop/layers/embeddings.py

```diff
@@ -303,8 +303,49 @@ def apply_2d_rotary_pos_emb(t, freqs_x, freqs_y):
 
         return torch.cat((apply_rotary_pos_emb(t_x, freqs_x),
                           apply_rotary_pos_emb(t_y, freqs_y)), dim=-1)
+
+class LegacySinusoidalEmbedding(Embedding):
```

Comment:
Let's not call it Legacy. We should have just one SinusoidalEmbedding, with options if needed.

Reply:
Removed post ablation